modules/sc-mesh-secure-deployment/src/nats/src/comms_controller.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from . import comms_command
+from . import comms_settings
+from . import comms_status
+from . import comms_mesh_telemetry
+
+
+class CommsController:  # pylint: disable=too-few-public-methods
+    """
+    Mesh network
+    """
+
+    def __init__(self, interval: int = 1000):
+        self.interval = interval
+        self.debug_mode_enabled = False
+
+        # base logger for comms and which is used by all other modules
+        self.main_logger = logging.getLogger("comms")
+        self.main_logger.setLevel(logging.DEBUG)
+        log_formatter = logging.Formatter(
+            fmt="%(asctime)s :: %(name)-18s :: %(levelname)-8s :: %(message)s"
+        )
+        console_handler = logging.StreamHandler()
+        console_handler.setFormatter(log_formatter)
+        self.main_logger.addHandler(console_handler)
+
+        self.c_status = []
+        # TODO how many radios?
```

Comment:
I believe at some point this will be determined by a configuration file retrieved from the MDM server, but it is possible to get the radios from the system:
```sh
$ awk -F= '$2=="wlan"{getline;print $2}' /sys/class/net/*/uevent
# wlan1
# wlp1s0
```

Reply:
SECO-3162